.gitignore

```diff
@@ -148,5 +145,3 @@ cython_debug/
 .local
 *.DS_Store
 
-# Generated html files from docs 
-docs/generated/*
```

Comment:
For reference: I had been generating some files from a template before everything was automated, following the pattern set in the xarray docs. I ended up deleting this, so this line in the gitignore is no longer neccessary.